docs/guides/fractional-gates.ipynb

```diff
@@ -0,0 +1,365 @@
+    "\n",
+    "from qiskit_ibm_runtime import QiskitRuntimeService\n",
+    "\n",
+    "\n",
+    "num_qubits = 5\n",
+    "num_time_steps = 3\n",
+    "rx_angle = 0.1\n",
+    "rzz_angle = 0.1\n",
+    "\n",
+    "ising_circuit = QuantumCircuit(num_qubits)\n",
+    "for i in range(num_time_steps):\n",
+    "    # rx layer\n",
+    "    for q in range(num_qubits):\n",
+    "        ising_circuit.rx(rx_angle, q)\n",
+    "    for q in range(1, num_qubits-1, 2):\n",
+    "        ising_circuit.rzz(rzz_angle, q, q+1)\n",
+    "    # 2nd rzz layer\n",
+    "    for q in range(0, num_qubits-1, 2):\n",
+    "        ising_circuit.rzz(rzz_angle, q, q+1)\n",
+    "    ising_circuit.barrier()\n",
+    "ising_circuit.draw('mpl')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "09ef3ed0-7126-45a2-a876-35a4f540c949",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Specify a backend which has fractional gates enabled, allowing a transpiler pass manager to take that into its own context\n",
+    "service = QiskitRuntimeService()\n",
+    "backend_fractional = service.least_busy(use_fractional_gates=True)\n",
+    "backend_conventional = service.least_busy(use_fractional_gates=False)\n",
+    "\n",
+    "pm_fractional = generate_preset_pass_manager(optimization_level=3, backend=backend_fractional, scheduling_method=\"alap\")\n",
+    "pm_conventional = generate_preset_pass_manager(optimization_level=3, backend=backend_conventional, scheduling_method=\"alap\")\n",
+    "\n",
+    "ising_circuit_fractional = pm_fractional.run(ising_circuit)\n",
+    "ising_circuit_conventional = pm_conventional.run(ising_circuit)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0013f5fa-4072-4aa4-94fe-7e195435f828",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Draw timeline of circuit with conventional gates\n",
+    "draw_timeline(ising_circuit_conventional, idle_wires=False, time_range=(0, 500), style=IQXSimple())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "08dd1cdf-8b34-47c2-8324-f3538c9d1ab6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Draw timeline of circuit with fractional gates\n",
+    "draw_timeline(ising_circuit_fractional, idle_wires=False, time_range=(0, 500), style=IQXSimple())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "48bad5e6-3c7a-4bd0-b050-e440c3257be2",
+   "metadata": {},
+   "source": [
+    "## Where to use fractional gates\n",
+    "\n",
+    "Historically, the basis gates available on QPUs in the fleet of IBM Quantum hardware have been **`ECR`**, **`X`**, **`RZ`**, **`SX`**, and **`ID`** which creates a constraint on circuits with single and two-qubit rotations. For example, an $R_X(\\theta)$ gate, when transpiled to a QPU in the IBM Quantum fleet, must decompose into a series of $RZ$ and $\\sqrt{X}$ gates -- creating a circuit with a depth of five instead of one.\n",
```

Comment:
RZ is virtual-Z gate and zero cost. Here you can replace two physical pulses implementing SX gates with one pulse.